README.md

```diff
@@ -54,7 +54,7 @@ sudo apt install libavutil-dev libavcodec-dev libavformat-dev libswscale-dev lib
 
 #### Fedora
 ```
-$ sudo dnf install wayland-devel wayland-protocols-devel ffmpeg-free-devel
+$ sudo dnf install wayland-devel wayland-protocols-devel ffmpeg-free-devel pulseaudio-libs-devel
```

Comment:
Seems like this doesn't belong in this commit.

Reply:
Dropped and moved to #203